project/Build.scala

```diff
@@ -30,6 +30,7 @@ import scala.util.Properties.isJavaAtLeast
 
 object Build {
   val scalacVersion = "2.12.8"
+  val referenceVersion = "0.13.0-bin-20190213-d7090ab-NIGHTLY"
```

Comment:
We'll need a more recent nightly, because of quotes in StagedTuple and https://github.com/lampepfl/dotty/commit/b32a4a8e96d7e2efbfe276605291db8a3f8f69d1